enterprise/api/v1/views.py

```diff
@@ -312,14 +313,16 @@ def enroll_learners_in_courses(self, request, pk):
             existing_users = {
                 result.pop('user') for result in results['successes'] if result['course_run_key'] == course_run
             }
-            LOGGER.info("Successfully bulk enrolled learners: {}".format(pending_users | existing_users))
-            if serializer.validated_data.get('notify'):
-                enterprise_customer.notify_enrolled_learners(
-                    catalog_api_user=request.user,
-                    course_id=course_run,
-                    users=pending_users | existing_users,
-                    admin_enrollment=True,
-                )
+            if len(pending_users | existing_users) > 0:
```

Comment:
Assuming this is to prevent the empty log lines, nice!

Reply:
yes that is this bit!